js/main.js

```diff
@@ -1,2 +1,40 @@
 /* global data */
 /* exported data */
+
+var entryArray = [];
+var nextEntryId = 0;
```

Comment:
The data model is already in `data.js`

Reply:
entryArray and nextEntryId were removed from main.js